trieste/models/gpflux/models.py

```diff
@@ -24,9 +22,9 @@
 from ...data import Dataset
 from ...types import TensorType
 from ..interfaces import TrainableProbabilisticModel
-from ..optimizer import BatchOptimizer
+from ..optimizer import KerasOptimizer
 from .interface import GPfluxPredictor
-from .utils import sample_dgp
+from .sampler import sample_dgp
```

Comment:
Not due to your change, but the samplers in sampler.py (previously utils.py) should really have docstrings.